frontend/components/bc/input/BcInputError.vue

```diff
@@ -0,0 +1,32 @@
+<script lang="ts" setup>
+const idError = useId();
+  /**
+   * Spacing of error message will be removed by explicitly passing `false`.
+   * This should encourage to always think about the error message.
+   */
+export type BcInputError = false | string;
+defineProps<{
+  error?: BcInputError;
+}>();
+</script>
+
+<template>
+  <slot
+    :id-error
+    :aria-invalid="!!error"
+    :aria-describedby="idError"
+  />
+  <div v-if="error !== false" :id="idError" class="bc-input-error">
+    {{ error }}
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.bc-input-error {
+  color: var(--text-color--error);
+  grid-column: span 2;
+  margin-left: auto; // 1.
+  --line-height--default: 1.2;
```

Comment:
this is used instead of a comment and should tell you:

We are setting the `default line-height` here. (<- which is a `code smell`)
Should be done globally.

Comments have to be maintained additionally. Code that documents itself hasn't.